File: frontend/pages/product.php
<!-- Egy könyv oldala -->
<div class="product">
    <figure>
        <img class="product-book" src="<?php echo Konyvklub\get_image($actual_book->image); ?>" alt="<?php echo $actual_book->title; ?>">
    </figure>
    <div class="description">
        <h4><?php echo $actual_book->author; ?></h4>
        <h2><?php echo $actual_book->title; ?></h2>
        <span class="product-price"><?php echo $actual_book->price; ?> Ft</span>

        <div class="product-card">
            <form action="" method="post" onsubmit="return loggedIn()">
                <button type="submit" name="cart_add" value="<?php echo $actual_book->isbn; ?>">Kosárba teszem</button>
            </form>
        </div>
        <hr>
        <h5>Leírás:</h5>
        <p><?php echo $actual_book->description; ?></p>

        <!-- <iframe width="700" height="315" src="https://www.youtube.com/embed/WRp-lYtNc_Q" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe> -->
    </div>
</div>

File: KonyvklubAdmin/KonyvklubAdmin/Models/Product.cs
namespace KonyvklubAdmin.Models
{
    public class Product
    {
        public string isbn { get; set; }
        public string title { get; set; }
        public string author { get; set; }
        public string description { get; set; }
        public string category { get; set; }
        public int price { get; set; }
        public int stock { get; set; }
        public int ordered { get; set; }
        public string image { get; set; }

        public Product() { }

        public Product(string[] line)
        {
            isbn = line[0];
            title = line[1];
            author = line[2];
            description = line[3];
            category = line[4];
            price = int.Parse(line[5]);
            stock = int.Parse(line[6]);
            ordered = int.Parse(line[7]);
            image = line[8];
        }
    }
}
